### DISCUSSION TITLE
Search box with greater or less than

### DISCUSSION THREAD

**Discussion by mrmarshallmark** [microsoft-search/pnp-modern-search#3322] [Q&A]
Is it possible to search upon a list item (which is of type 'number') where the search criteria is conditioned by an operator like greater or less than.  For example if I have a column in a list named 'SQFT' and the column contains a numbers (1000, 2000, 3000..) Can I get results that are larger than 2000 only.

SQFT:({searchTerms})path:"https://nameofsite.sharepoint.com/sites/Brokerage/Lists*"  -ContentClass=STS_ListItem

**Comment by kasperbolarsen** [comment]
You absolutely can. The only tricky part is to map the crawled property. I can't remember if you can create a new managed property of type Integer, but you can always map one of the RefinableInt to the crawled property.  This will enable you to have a simple query like RefinableInt01 > 2000

**Comment by mrmarshallmark** [comment]
I'd want it to return all records which are > the amount in SQFT that the user inputs into the Search box, but i'm not sure of the syntax and where to put the greaterr than symbol
SQFT>({searchTerms})
or something like
(SQFT({searchTerms})>SQFT({searchTerms}))) not really sure any suggestions i could try?

**Comment by kasperbolarsen** [comment]
The first option looks just fine. If in doubt I will recommend using either SP Editor or SharePoint Search Query tool to verify your data and queries.
